interface/login/login.php

```diff
@@ -238,7 +240,7 @@ function getLanguagesList(): array
     'googleSigninClientID' => $GLOBALS['google_signin_client_id'],
     'logoArea' => $logoarea,
     'displayExtraLogo' => $GLOBALS['extra_logo_login'],
-    'primaryLogoSrc' => file_get_contents($GLOBALS["images_static_absolute"] . "/login-logo.svg"),
+    'primaryLogoSrc' => file_get_contents($loginlogolocation),
```

Comment:
If we move the main logos into this directory then many upgraders won't get updated logos in future.
Rec:
1. keeping main logos where they originally were in public and use them by default
2. use the logo at `$GLOBALS["images_static_absolute"] . "/login-logo.svg"` if it exists (also put a comment above this line describing what it does; ie. if a user placed a file at `site/"site_id"/images/login-logo.svg`, then it will override the logo on the logon screen)

Reply:
The only way to know it exists is that there has to be an entry of that file name.